Tasks/PowerShellV2/powershell.ts

```diff
@@ -7,6 +7,137 @@ import { validateFileArgs } from './helpers';
 import { ArgsSanitizingError } from './errors';
 import { emitTelemetry } from 'azure-pipelines-tasks-utility-common/telemetry';
 var uuidV4 = require('uuid/v4');
+import * as msal from "@azure/msal-node";
```

Comment:
special handling is needed for older node runners (node10 needs msal1).  We'll need to ensure we add that to test matrix